### DISCUSSION TITLE
part-ii/2017-58/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#65] [Question comments]
Part II 2017 Fluid Dynamics II: Paper 1, Section II, B 

https://questions.tripos.org/part-ii/2017-58/

**Comment by acornnroca** [comment]
<details><summary>[answer]</summary>

$$U ~ x^{1/5}$$

$$\delta ~ x^{2/5}$$

then get the equation if you do it


BCs are 

$$f'(0) f''(0) = -1

no penetration 

$$u = 0 at y \to \infinity$$

</details>

**Comment by acornnroca** [comment]
<details><summary>[spoiler title]</summary>

to determine the sign of the transverse flow, consider the flux

it grows

so we've got entrainment so v negative

</details>
